docs/reference/health/health.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[health-api]]
+=== Health API
+++++
+<titleabbrev>Health</titleabbrev>
+++++
+
+An experimental API that returns the health status of an {es} cluster.
+
+This API is currently experimental for internal use by Elastic software only.
+
+[[health-api-request]]
+==== {api-request-title}
+
+`GET /_internal/_health`
+
+[[health-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `monitor` or
+`manage` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+[[health-api-desc]]
+==== {api-description-title}
+
+The health API returns a the health status of an Elasticsearch cluster. It
+returns a list of components that compose Elasticsearch functionality. Each
+component's health is determined by health indicators associated with the
+component.
+
+Each indicator has a health status of: `green`, `yellow` or `red`. The indicator will
+provide an explanation and metadata describing the reason for its current health status.
+
+A components status is controlled by the worst indicator status. The cluster's status
+is controlled by the worst component status.
+
+[[health-api-query-params]]
+==== {api-query-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=local]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+[[health-api-response-body]]
+==== {api-response-body-title}
+
+`cluster_name`::
+    (string) The name of the cluster.
+
+`status`::
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=cluster-health-status]
+
+`number_of_nodes`::
+    (integer) The number of nodes within the cluster.
+
+`number_of_data_nodes`::
+    (integer) The number of nodes that are dedicated data nodes.
```

Comment:
I wonder if we need this initially? I think we would want a map by role or something instead to give an accurate summary of the cluster. We would also want to maybe know the status compared to the desired nodes.

Reply:
I deleted all the metadata and we can have a PR just dedicated to that.